src/main/java/frc/robot/subsystems/climb/ClimbIOVictorSPX.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems.climb;
+
+import com.ctre.phoenix.motorcontrol.VictorSPXControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+import frc.robot.Constants;
+
+public class ClimbIOVictorSPX implements ClimbIO
+{
+    public final VictorSPX           _climbVictorSPXLeft;
+    public final VictorSPX           _climbVictorSPXRight;
+    public final AnalogPotentiometer _potentiometerLeft;
+    public final AnalogPotentiometer _potentiometerRight;
+    public final double              _leftOffset  = Constants.Climb.LEFT_ZERO_OFFSET;
+    public final double              _rightOffset = Constants.Climb.RIGHT_ZERO_OFFSET;
```

Comment:
These two variables might change if we reconfigure them through the dashboard, so we won't want them to be final:

```suggestion
    private double                    _leftOffset  = Constants.Climb.LEFT_ZERO_OFFSET;
    private double                    _rightOffset = Constants.Climb.RIGHT_ZERO_OFFSET;
```